testing/test_acquisition.py

```diff
@@ -145,6 +163,34 @@ def test_PoF_validity(self):
         self.assertTrue(np.allclose(self.acquisition.evaluate(X2), 0), msg="Right half of plane is not feasible")
 
 
+class TestLowerConfidenceBound(_TestAcquisition, unittest.TestCase):
+    def setUp(self):
+        super(TestLowerConfidenceBound, self).setUp()
+        self.model = self.create_plane_model()
+        self.acquisition = GPflowOpt.acquisition.LowerConfidenceBound(self.model, 3.2)
+
+    def test_objective_indices(self):
+        self.assertEqual(self.acquisition.objective_indices(), np.arange(1, dtype=int),
+                         msg="LCB returns all objectives")
+
+    def test_object_integrity(self):
+        super(TestLowerConfidenceBound, self).test_object_integrity()
+        self.assertEqual(self.acquisition.sigma, 3.2)
+
+    def test_LCB_validity(self):
+        design = GPflowOpt.design.RandomDesign(200, self.domain).generate()
+        p = self.model.predict_f(design)[0]
+        q = self.acquisition.evaluate(design)
+        np.testing.assert_array_less(q,p)
+
+    def test_LCB_validity_v2(self):
```

Comment:
v2 implies this is the next version for me (thinking that the other one should be removed). Maybe just add just _2.